Documentation/project-docs/developer-guide.md

```diff
@@ -38,21 +37,38 @@ the managed build which produces the MSIL code and nuget packages that make up C
 Calling the script `build` attempts to build both the native and managed code.
 Only use it when the parameters that you are passing to the script apply for both components. Otherwise, use the scripts `build-native` and `build-managed` respectively.
 
-**Examples**
+Most of the build configurations are defaulted based on where you are building but here are a few common parameters to customize the build configuration:
 
-- Building in debug mode for platform x64
+- `-framework` identifies the target framework for the build. It defaults to `netcoreapp` but possible values include `netcoreapp`, `netfx` or `uap`. (msbuild property `TargetGroup`)
```

Comment:
Is it worth to add explanation of netfx=Desktop / Full .NET Framework and uap=UWP, .NET Native? (Assuming I got uap right)